slides/02-data-budget.qmd

```diff
@@ -219,33 +219,34 @@ We will use this tomorrow
 
 # Exploratory data analysis for ML 🧐
 
-## 
+## Your turn {transition="slide-in"}
+
+![](images/parsnip-flagger.jpg){.absolute top="0" right="0" width="150" height="150"}
+
+*Explore the `frog_train` data on your own!*
+
+* *What's the distribution of numeric variables like age?*
```

Comment:
```suggestion
* *What's the distribution of the outcome, latency?*
* *What's the distribution of numeric variables like age?*
```

This might be a useful prompt if we are going to use this hands-on as a motivator for stratification on latency (especially since it is the first image on the next slide)